scripts/fault-domain-detect.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -o nounset -o errexit
```

Comment:
Why was it removed?
Actually it will fail if something goes wrong within the script execution. Isn't this what we would expect?